src/kind.ts

```diff
@@ -70,15 +70,20 @@ export function getKindConfig(): KindConfig {
 
 // this action should always be run from a Linux worker
 export async function downloadKind(version: string) {
-    let url: string = `https://github.com/kubernetes-sigs/kind/releases/download/${version}/kind-linux-amd64`;
-    console.log("downloading kind from " + url);
-    let downloadPath: string | null = null;
-    downloadPath = await tc.downloadTool(url);
-    const binPath: string = "/home/runner/bin";
-    await io.mkdirP(binPath);
-    await exec.exec("chmod", ["+x", downloadPath]);
-    await io.mv(downloadPath, path.join(binPath, "kind"));
+    let toolPath: string = tc.find("kind", version);
```

Comment:
Is this because the tool has already been downloaded on the runner?

Then I would much rather make the check outside of `downloadKind` - what do you think?

Reply:
Actually, in this case, it might even make sense to expose this as a config for the action?

Reply:
> Then I would much rather make the check outside of downloadKind - what do you think?

It is better. I'll change it that way.

Reply:
Generally, downloaded tools in actions are cached and reused (e.g., actions/setup-go), so I feel that there is no need to expose it as a config. Is my understanding correct?

Reply:
> Then I would much rather make the check outside of downloadKind - what do you think?

I did it at 76e5e91.